subprojects/docs/src/samples/credentials-handling/publishing-credentials/groovy/build.gradle

```diff
@@ -18,11 +18,18 @@ publishing {
     }
 // tag::repositories[]
     repositories {
+        // using the repository name to resolve credentials
         maven {
             name = 'mySecureRepository'
             credentials(PasswordCredentials)
             // url = uri(<<some repository url>>)
         }
+        // using an explicit identity to resolve credentials
+        maven {
+            name = 'myAwesomeReleaseRepository'
```

Comment:
I would be tempted to use a name with spaces as it seems to be a more common scenario, i.e. `My Awesome Repository (Release)`. However, this illustrates well the goal of specifying the identity.